src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random colorRandom = new Random();
+    private int colorInt = colorRandom.nextInt(Colors.values().length);
```

Comment:
If you declare your `colorInt` variable at class-level it will be the same value in `getRandomColor()` method on each call. You should declare this variable at method-level (local).